components/style/values/specified/position.rs

```diff
@@ -486,3 +490,179 @@ impl From<GridAutoFlow> for u8 {
         result
     }
 }
+
+#[cfg_attr(feature = "gecko", derive(MallocSizeOf))]
+#[derive(Clone, Debug, PartialEq)]
+/// Single template area
```

Comment:
Hmm, this is not a single area, right? Also, this would probably benefit from a spec link

Reply:
Hmm, yeah, it's not real single but belongs to each `grid-template-areas`; thus, that's why I documented it with `Single`.
But I think you're right, this wording might have misleading.
Just let me document it with the spec link! Thanks!